### PR TITLE
[VL] package and velox scripts add alinux support

### PR DESCRIPTION
**Pull request by liujiayi771**
## What changes were proposed in this pull request?

package and load so from jar support alinux

### CODE REVIEW

**Comment by liujiayi771** [comment]
@zhejiangxiaomai Can you help review?

**Review by zhejiangxiaomai — APPROVED** [review]
Thanks.

File: dev/package.sh [hunk 1]
```diff
@@ -45,5 +45,10 @@ elif [ "$LINUX_OS" == "centos" ]; then
     process_setup_centos_7
   fi
 fi
+elif [ "$LINUX_OS" == "alinux" ]; then
```
Comment (zhouyuan) [inline dev/package.sh:48]: the format seems wrong here:

```
./dev/package.sh: line 47: syntax error near unexpected token `elif'
./dev/package.sh: line 47: `elif [ "$LINUX_OS" == "alinux" ]; then'
Build step 'Execute shell' marked build as failure
```

Comment (zhejiangxiaomai) [reply dev/package.sh:48]: Fire a Pr to fix it. https://github.com/oap-project/gluten/pull/1342

**Pull request merged**